pkg/sql/testdata/logic_test/aggregate

```diff
@@ -12,7 +12,7 @@ CREATE TABLE kv (
 query IIIIRRRRBB
 SELECT MIN(1), MAX(1), COUNT(1), SUM_INT(1), AVG(1), SUM(1), STDDEV(1), VARIANCE(1), BOOL_AND(true), BOOL_AND(false) FROM kv
```

Comment:
It's strange that `SUM_INT` now behaves differently than `SUM`. Also, a lot of these are inconsistent with classic SQL. Are we aware of this, and is it intentional?

```
postgres=# SELECT SUM(1);
 sum
-----
   1
(1 row)
```